src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -1159,7 +1165,7 @@ static void Main()
         int y;
 
         (x, x, x, x, x, x, x, x, x, y) = (1, 1, 1, 1, 1, 1, 1, 1, 4, 2);
-        System.Console.WriteLine(x + "" "" + y);
+        System.Console.WriteLine(string.Concat(x, "" "", y));
```

Comment:
This one is interesting. By calling `x.ToString()`, I seem to have triggered some strange behaviour in the optimizer. See [this SharpLab](https://sharplab.io/#v2:D4AQDABCCMDcCwAoEBmKAmCBhCBvJEhUaIALBALIAUAlHgUYwDYD2AdgOYQAeCijjAJZsALhACefAUSrcANDwXzFK5WqULxdALwQq0BQYhGThs8YWkF6GlOkQG0mAE5ZEANQQARF48SAdAAqLADKIgBOwhy0to4QAL5xcahQ5ABytPT89qycPHbSwmKScYyyGqoV6pUSOnqmFo0NDVYQNgUCpUQuskGhEVGZnj5+4n1hkZwxHYmI8UA=) for a demonstration.

To avoid triggering it in this test, I changed it to explicitly call `string.Concat`. I'm happy to change this approach.